openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -63,6 +63,7 @@
     .wrapper--content {
       padding: $baseline-v ($baseline-h/2);
       background-color: $bg-content;
+      color: $copy-color;
```

Comment:
A lot of the content in the staff tools was too pale. I used the accessible $copy-color that is used in the student assessments.

Before:
![image](https://cloud.githubusercontent.com/assets/484484/20154174/22a0b014-a694-11e6-8f8e-e9c9049cef97.png)

After:
![image](https://cloud.githubusercontent.com/assets/484484/20154192/3142a474-a694-11e6-98c6-605a03c10e77.png)